docs/source/spec-test-format.rst

```diff
@@ -31,25 +35,116 @@ A Test Scenario File has the following keys:
+      Default is 90 seconds.
+
+    This operation is undefined and MUST NOT be used when the deployment is
+    a sharded cluster.
+
+    Example::
+    
+      assertPrimaryRegion:
+        region: US_WEST_1
+        timeout: 15
+    
+  * sleep: do nothing for the specified duration. The value MUST be the duration
+    to sleep for, in seconds.
+
+    Example::
+    
+      sleep: 10
+    
+  * waitForIdle: wait for cluster maintenance state to become "idle".
+    The value MUST be ``true``.
+    
+    The workload executor MUST ignore the value of this key, so that
+    the value can be changed to a hash in the future to provide options
+    to the operation.
+
+    Example::
 
-  * uriOptions (document): Document containing ``key: value`` pairs of URI options that must be included in the
-    connection string passed to the workload executor by the *Test Orchestrator*.
+      waitForIdle: true
 
-* driverWorkload (document): Object describing a *Driver Workload*. Has the following keys:
+* driverWorkload (document): Description of the driver workload to execute
+  The document must be a complete test as defined by the
+  `Unified Test Format specification <https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst>`_.
+  
+  The workload MUST use a single test, as defined in the unified test format
+  specification.
+  
+  The workload MUST use the ``loop`` unified test format operation to
+  define the MongoDB operations to execute during maintenance. There MUST
+  be exactly one ``loop`` operation per scenario, and it SHOULD be the last
+  operation in the scenario.
 
-  * collection (string): Name of the collection to use for running test operations.
-  * database (string): Name of the database to use for running test operations.
-  * testData (array, optional): Array of documents to be inserted into the ``database.collection`` namespace before
-    starting the test run. Test data insertion is performed by the *Test Orchestrator* and this field MUST be ignored
-    by the *Workload Executor*.
-  * operations (array): Array of Operation objects, each describing an operation to be executed. The operations are run
-    sequentially and repeatedly until the maintenance completes. Each object has the following keys:
+  The scenario MUST use ``storeErrorsAsEntity``, ``storeFailuresAsEntity``,
+  ``storeSuccesesAsEntity`` and ``storeIterationsAsEntity`` operation arguments
+  to allow the workload executor to retrieve errors, failures and operation
+  counts for the executed workload. The entity names for these options MUST
+  be as follows:
+  
+  - ``storeErrorsAsEntity``: ``errors``
+  - ``storeFailuresAsEntity``: ``failures``
+  - ``storeSuccessesAsEntity``: ``successes``
+  - ``storeIterationsAsEntity``: ``iterations``
+  
+  The scenario MAY use ``storeEventsAsEntities`` operation argument
+  when defining MongoClients to record CMAP events published during maintenance.
```

Comment:
Should this be changed in light of https://github.com/mongodb/specifications/pull/903/files#r574281445?

Reply:
Added the command monitoring bits in.